charts/security-apps/Chart.yaml

```
@@ -3,8 +3,8 @@ name: security-apps
 description: Argo CD app-of-apps config for security applications
 type: application
 # version and appVersion are in sync in this chart!
-version: 0.13.0
-appVersion: 0.13.0
+version: 1.0.0
+appVersion: 1.0.0
```

```suggestion
version: 0.14.0
appVersion: 0.14.0
```

breaking change rule for major bumps only applies to 1.x and higher versions.

this chart is still using 0.x for that exact purpose and should not get released as stable 1.x until we are sure that we have tried and tested all aspects of the chart and it's contents.

---

Wow, didn't know that. TIL. Thanks